templates/android-dot-com/index.html

```diff
@@ -86,7 +86,7 @@
         </div>
       </div>
 
-      <div class="android-drawer mdl-layout__drawer">
+      <div class="android-drawer mdl-layout__drawer" aria-hidden="true">
```

Comment:
The most screen readers do not need valid markup on load. The attribut can be added at runtime. I personally prefer valid markup on load, but thats your decision.